### DISCUSSION TITLE
Error when creating front ports

### DISCUSSION THREAD

**Discussion by tinkerberlin** [netbox-community/netbox#12615] [Help Wanted!] [answered]
Hi there, 
I'm trying to create a patch panel in Netbox. I've created 24 rear ports (named "Hinten[1-24]" and now am trying to create 24 matching front ports (named "Vorne[1-24]"). 
![image](https://github.com/netbox-community/netbox/assets/99590368/37b9b527-2afb-4ad0-99a5-6f0004e9e928)

However, when I do this, and hit "create" I get the following error: 
![image](https://github.com/netbox-community/netbox/assets/99590368/b543e5d9-a979-460a-98c7-86b4269ce183)

When I try and create a single front port (e.g. "Vorne1") that works without a problem. 

Any idea what I'm doing wrong?

Thanks!!

**Comment by candlerb** [comment, marked as answer]
It's a bug in Netbox that it barfs with an exception, it should give you a more friendly error.

But I think you'll need to select *all 24 rear ports* into the box labelled "rear ports", since you created 24 separate rear ports (and presumably you selected each rear port has having 1 position). i.e. Hinten1:1, Hinten2:1, Hinten3:1 etc

The other way to do this is to create 1 rear port with 24 positions, and then you would select the 24 positions of the same rear port, i.e. Hinten1:1, Hinten1:2, Hintent1:3 etc.  That's if you want to run a single cable (like an MPO cable) to a similar ODF with 1 rear port and 24 front ports.

**Reply by tinkerberlin** [reply]
Thanks! Yes - that seems to have fixed the problem.
